src/main/groovy/com/github/jengelman/gradle/plugins/shadow/transformers/ApacheNoticeResourceTransformer.groovy

```diff
@@ -34,34 +36,48 @@ import java.text.SimpleDateFormat
  * @author John Engelman
  */
 class ApacheNoticeResourceTransformer implements Transformer {
+
+    @Input
     Set<String> entries = new LinkedHashSet<String>()
```

Comment:
```suggestion
    private Set<String> entries = new LinkedHashSet<String>()
```

It seems like this is modified while running the transformer, so I think this should be made private.